src/components/FlowRunTimelineSubFlowRunDetails.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+  <template v-if="subFlowRun">
+    <page-heading :crumbs="crumbs" size="sm" class="mb-2">
+      <template #actions>
+        <slot />
+      </template>
+    </page-heading>
```

Comment:
Is the page heading component what you're looking for here? I'd expect that to be used solely in the context of a page and not an aside; maybe this is ok, just not something I've seen done before and could have unexpected results if we do make any app-wide changes to headings

Reply:
Ok, swapped this for `p-heading`